projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -752,6 +752,13 @@ public static function enqueue_block_editor_assets() {
 			);
 		}
 
+		// We need this information in the ai-chat feature, because it cant be activated if instant search is not enabled.
+		if ( Jetpack::is_module_active( 'search' ) ) {
```

Comment:
Better to use the Search_Plan()->supports_search() package method which also works in wpcom. https://github.com/Automattic/jetpack/blob/273fb4c4e309c1abd2698921ce6ca9669e6dc2a6/projects/plugins/jetpack/_inc/lib/class-jetpack-ai-helper.php#L101

Reply:
Done